bin/load-mocks

```diff
@@ -564,6 +564,43 @@ def main(num_events=1, extra_events=False):
                         user=generate_user(),
                     )
 
+            if load_transactions:
+                for day in range(14):
+                    for hour in range(24):
+                        timestamp = timezone.now() - timedelta(days=day, hours=hour)
+                        transaction_user = generate_user()
+                        transaction1 = create_sample_event(
+                            project=project,
+                            platform="javascript-transaction",
+                            event_id=uuid4().hex,
+                            user=transaction_user,
+                            timestamp=timestamp,
+                            # start_timestamp decreases based on day so that there's a trend
+                            start_timestamp=timestamp
+                            - timedelta(
+                                milliseconds=int(random.normalvariate(2000 - 50 * day, 250))
+                            ),
+                            measurements={
+                                "fp": {"value": max(random.normalvariate(2000, 200), 1000)},
+                                "fcp": {"value": max(random.normalvariate(2250, 200), 1000)},
+                                "lcp": {"value": max(random.normalvariate(2800, 400), 2000)},
+                                "fid": {"value": max(random.normalvariate(5, 2), 1)},
+                            },
+                        )
+                        create_sample_event(
+                            project=project,
+                            platform="transaction",
+                            event_id=uuid4().hex,
+                            user=transaction_user,
+                            timestamp=timestamp,
+                            # start_timestamp increases based on day so that there's a trend
+                            start_timestamp=timestamp
+                            - timedelta(
+                                milliseconds=int(random.normalvariate(2000 + 50 * day, 250))
+                            ),
+                            trace=transaction1.get_tag("trace"),
```

Comment:
Do you need to randomize the trace ids?

Reply:
Hm I'll add a comment here, but I'm creating a "frontend" transaction with `javascript-transactions.json` on L572, getting its random trace id, and then setting that here on the "backend" transaction so that we can follow the trace:
<img width="1735" alt="image" src="https://user-images.githubusercontent.com/4205004/95597626-e0b12580-0a1c-11eb-9b81-ae05c7fc7d36.png">